static/sass/styles.scss

```diff
@@ -359,6 +358,27 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
   }
 }
 
+.p-side-navigation__drawer.is-documentation{
+ ul li {
+  background: #f3f3f3;
+
+  a {
+    color: #3097ff;
+    padding-left: 0;
+
+    &:visited {
+      color: #AB83D3;
+    }
+  }
+ }
+}
```

Comment:
We should probably use a different pattern altogether but if we're going forward with this let's use vanilla colours!